doc/api/http.md

```diff
@@ -488,7 +488,7 @@ const proxy = createServer((req, res) => {
 });
 proxy.on('connect', (req, clientSocket, head) => {
   // Connect to an origin server
-  const { port, hostname } = new URL(`http://${req.url}`);
+  const { port, hostname } = new URL(`http://${req.headers.host}${req.url}`);
```

Comment:
This is wrong, as the result would be something like `http://localhost:49569www.google.com:80` instead of `http://www.google.com:80`.

Reply:
`http://localhost:49569www.google.com:80` is not a valid URL you cannot do a request on this it is missing a slash
`http://localhost:49569/www.google.com:80` you can test it with curl.

and no the result is not wrong, just tested

Reply:
The result here should be `http://www.google.com:80`. It's a proxy, you want to connect to the upstream server specifies in the `path` option.

Reply:
oh sorry about that you are right on this one @lpinca